homeassistant/components/home_connect/coordinator.py

```diff
@@ -272,6 +276,13 @@ async def _get_appliance_data(
         appliance_data_to_update: HomeConnectApplianceData | None = None,
     ) -> HomeConnectApplianceData:
         """Get appliance data."""
+        self.device_registry.async_get_or_create(
+            config_entry_id=self.config_entry.entry_id,
+            identifiers={(DOMAIN, appliance.ha_id)},
+            manufacturer=appliance.brand,
+            name=appliance.name,
+            model=appliance.vib,
```

Comment:
We can now simplify the device info in `entity.py` and just pass the `identifiers` parameter since we create the device entry here with all the data, manufacturer, model etc.

Reply:
Cool! I didn't know we could do that